+++ /opt/cosbench/conf/.driver/config.ini
@@ -78,7 +78,8 @@
 plugins/cosbench-core@8\:start,\
 plugins/cosbench-driver@8\:start,\
 plugins/cosbench-core-web@8\:start,\
-plugins/cosbench-driver-web@8\:start
+plugins/cosbench-driver-web@8\:start,\
+plugins/cosbench-manta@7\:start
 osgi.clean=true
 osgi.noShutdown=true
 osgi.startLevel=8
+++ /opt/cosbench/conf/manta-config.xml
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<workload name="manta-sample" description="sample benchmark for manta">
+
+  <storage type="manta" config="" />
+
+  <workflow>
+
+    <workstage name="init">
+      <work type="init" workers="1" config="createContainer=true;containers=r(1,2)" />
+    </workstage>
+
+    <workstage name="prepare">
+      <work type="prepare" workers="1" config="containers=r(1,2);objects=r(1,50);sizes=c(64)KB" />
+    </workstage>
+
+    <workstage name="main">
+      <work name="main" workers="8" runtime="30">
+        <operation type="read" ratio="80" config="containers=u(1,2);objects=u(1,50)" />
+        <operation type="write" ratio="20" config="containers=u(1,2);objects=u(51,100);sizes=c(64)KB" />
+      </work>
+    </workstage>
+
+    <workstage name="cleanup">
+      <work type="cleanup" workers="1" config="containers=r(1,2);objects=r(1,100)" />
+    </workstage>
+
+    <workstage name="dispose">
+      <work type="dispose" workers="1" config="containers=r(1,2)" />
+    </workstage>
+
+  </workflow>
+
+</workload>
+++ /opt/cosbench/start-driver.sh
@@ -22,7 +22,7 @@
 SERVICE_NAME=driver
 VERSION=`cat VERSION`

-OSGI_BUNDLES="cosbench-log_${VERSION} cosbench-tomcat_${VERSION} cosbench-config_${VERSION} cosbench-http_${VERSION} cosbench-cdmi-util_${VERSION} cosbench-core_${VERSION} cosbench-core-web_${VERSION} cosbench-api_${VERSION} cosbench-mock_${VERSION} cosbench-ampli_${VERSION} cosbench-swift_${VERSION} cosbench-keystone_${VERSION} cosbench-httpauth_${VERSION} cosbench-s3_${VERSION} cosbench-librados_${VERSION} cosbench-scality_${VERSION} cosbench-cdmi-swift_${VERSION} cosbench-cdmi-base_${VERSION} cosbench-driver_${VERSION} cosbench-driver-web_${VERSION}"
+OSGI_BUNDLES="cosbench-log_${VERSION} cosbench-tomcat_${VERSION} cosbench-config_${VERSION} cosbench-http_${VERSION} cosbench-cdmi-util_${VERSION} cosbench-core_${VERSION} cosbench-core-web_${VERSION} cosbench-api_${VERSION} cosbench-mock_${VERSION} cosbench-ampli_${VERSION} cosbench-swift_${VERSION} cosbench-keystone_${VERSION} cosbench-httpauth_${VERSION} cosbench-s3_${VERSION} cosbench-manta cosbench-librados_${VERSION} cosbench-scality_${VERSION} cosbench-cdmi-swift_${VERSION} cosbench-cdmi-base_${VERSION} cosbench-driver_${VERSION} cosbench-driver-web_${VERSION}"

 OSGI_CONSOLE_PORT=18089

